packages/react-vapor/src/components/svg/SvgNames.ts

```diff
@@ -0,0 +1,455 @@
+export type SvgNames =
```

Comment:
This list is done manually? :eyes:

Reply:
for now (I copiied and pasted from here https://vaporqa.cloud.coveo.com/#/styles/icons/list), but we should be able to autogenerate the type from figma